tests/unit/conftest.py

```diff
@@ -62,3 +69,66 @@ def __create_member(
         return mocked_member
 
     return __create_member
+
+
+@pytest.fixture
+def create_db_user(mocker: MockerFixture):
+    def __create_db_user(
+        user_id: Optional[int] = None,
+        discord_user_id: Optional[str] = None,
+        discord_guild_id: Optional[str] = None,
+        is_mod: Optional[bool] = None,
+        temporary_points: Optional[int] = None,
+        permanent_points: Optional[int] = None,
+        last_infraction_timestamp: Optional[datetime] = None,
+        get_strike_points: Optional[int] = None,
+    ):
+        mocked_user = mocker.Mock(
+            spec=User,
+            user_id=user_id,
+            discord_user_id=discord_user_id,
+            discord_guild_id=discord_guild_id,
+            is_mod=is_mod,
+            temporary_points=temporary_points,
+            permanent_points=permanent_points,
+            last_infraction_timestamp=last_infraction_timestamp,
+            get_strike_points=mocker.Mock(return_value=get_strike_points),
+        )
+        return mocked_user
+
+    return __create_db_user
+
+
+@pytest.fixture
+def create_embed(mocker: MockerFixture):
+    def __create_embed():
+        mocked_user = mocker.Mock(spec=discord.Embed)
+        return mocked_user
+
+    return __create_embed
+
+
+# @pytest.fixture
+# def create_db_strike(mocker: MockerFixture):
+#     def __create_db_strike(
+#         strike_id: Optional[int] = None,
+#         user_id: Optional[int] = None,
+#         severity: Optional[enums.StrikeSeverity] = None,
+#         reason: Optional[str] = None,
+#         created_timestamp: Optional[datetime] = None,
+#         created_by: Optional[str] = None,
+#         last_edited_timestamp: Optional[datetime] = None,
+#         last_edited_by: Optional[str] = None,
+#     ):
+#         mocked_strike = mocker.Mock(spec=Strike)
+#         mocked_strike.strike_id = strike_id
+#         mocked_strike.user_id = user_id
+#         mocked_strike.severity = severity
+#         mocked_strike.reason = reason
+#         mocked_strike.created_timestamp = created_timestamp
+#         mocked_strike.created_by = created_by
+#         mocked_strike.last_edited_timestamp = last_edited_timestamp
+#         mocked_strike.last_edited_by = last_edited_by
+#         return mocked_strike
+
+#     return __create_db_strike
```

Comment:
let's either delete this or punt this to a later PR if we're going to leave this commented out